test/libsolidity/syntaxTests/types/mapping/library_array_argument_external.sol

```diff
@@ -3,4 +3,3 @@ library L {
     }
 }
 // ----
-// TypeError: (27-58): Type is required to live outside storage.
```

Comment:
Why did we have this error before?

Reply:
This https://github.com/ethereum/solidity/pull/4798/files#diff-9f2a45efaa7b75ab7ec620d3177cf01a is the commit that added it. Don't know if there was a reason or if this was just a bug.